boards/shields/link_board_can/link_board_can.overlay

```diff
@@ -13,13 +13,12 @@
 
 	mcp2515@0 {
 		compatible = "microchip,mcp2515";
-		spi-port-name = "SPI_1";
```

Comment:
I forgot to add this to my initial PR but after becoming aware that the 125kbit/s bus-speed was the common default value I thought it was a good time to add these two changes.  I had previously discussed with @karstenkoenig and he'd agreed that the spi-port-name property was invalid and should be removed.

Reply:
This has been fixed in PR #19680